GOVERNANCE.md

```diff
@@ -2,94 +2,171 @@
+#### Privileges
 
-**Responsibilities:**
+- All privileges of the [Contributor role](#level-1---contributor), plus...
+- `@maintainer` role on [Discord](https://astro.build/chat)
+- New name color on Discord: **blue**.
+- Invitation to the private #maintainers channel on Discord.
+- Invitation to the `maintainers` team on GitHub.
+- Ability to moderate Discord.
+- Ability to push branches to the repo (No more personal fork needed).
+- Ability to review GitHub PRs.
+- Ability to merge _some_ GitHub PRs.
+- Ability to vote on _some_ initiatives (see [Voting](#voting) below).
 
-- May request write access to relevant Astro projects.
-- GitHub: May work on public branches of the source repository and submit pull requests from that branch to the main branch.
-- GitHub: Must submit pull requests for all changes, and have their work reviewed by other members before acceptance into the repository.
-- GitHub: May merge some pull requests (see Managing Pull Requests)
+#### Responsibilities
 
-**Nomination:**
+- Participate in the project as a team player.
+- Bring a friendly, welcoming voice to the Astro community.
+- Be active on Discord, especially in the #support channel.
+- Triage new issues.
+- Review pull requests.
+- Merge some, non-trivial community pull requests.
+- Merge your own pull requests (once reviewed and approved).
 
-- A nominee will need to show a willingness and ability to participate in the project as a team player.
-- Typically, a nominee will need to show that they have an understanding of and alignment with the project, its objectives, and its strategy.
-- Nominees are expected to be respectful of every community member and to work collaboratively in the spirit of inclusion.
-- Have submitted a minimum of 10 qualifying significant contributions (see list above).
-- You can be nominated by any existing Contributor (L2 or above).
-- Once nominated, there will be a vote by existing Contributors (L3 or above) (see [voting rules](#voting)).
+#### Nomination
 
-It is important to recognize that this role is a privilege, not a right. That privilege must be earned and once earned it can be removed (in a vote by project Stewards). However, under normal circumstances this role exists for as long as the Contributor wishes to continue engaging with the project.
+To be nominated, a nominee is expected to already be performing some of the responsibilities of a Maintainer over the course of at least a couple of weeks. In the past, we have used "10 PRs" as a rough minimum for potential Maintainers, but there is no hard requirement.
 
-Inactive Contributors will have voting rights removed after a certain period of time, however they will always retain their status. Inactivity requirements will be specified in a later governance change.
+In some rare cases, this role may be revoked by a project Steward. However, under normal circumstances this role is granted for as long as the contributor wishes to engage with the project.
```

Comment:
New personal goal: get a Steward _named_ Steward. That would make me so happy

Reply:
Stewart the Steward!